.github/workflows/formatting-changes.yml

```diff
@@ -0,0 +1,49 @@
+name: Formatting changes
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  format:
+    runs-on: ubuntu-24.04
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+          ref: ${{ github.event.pull_request.head.ref }}
+          token: ${{ secrets.GIX_CREATE_PR_PAT }}
+
+      - name: Restore npm cache
+        uses: actions/cache@v4
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-format-npm-${{ hashFiles('package-lock.json', 'package.json') }}
+          restore-keys: |
+              ${{ runner.os }}-format-npm-
+      
+      - name: Prepare
+        uses: ./.github/actions/prepare
+
+      - name: Run format Script
+        run: npm run format
+
+      - name: Check for Changes
+        id: check_changes
+        run: |
+          if [[ -n "$(git status --porcelain)" ]]; then
+            echo "changes_detected=true" >> $GITHUB_OUTPUT
+          else
+            echo "changes_detected=false" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Commit format
+        if: steps.check_changes.outputs.changes_detected == 'true'
+        uses: EndBug/add-and-commit@v9
+        with:
+          add: '.'
+          message: '🤖 Apply formatting changes'
+          push: true
+          pull_strategy: 'default'
```

Comment:
`pull` instead of `pull_strategy`?

Reply:
removed in recent commit. pull not necessary since PRs and PR commits should always be on latest state of branch.

Reply:
so, will it be removed here too?